lib/operations/connect.js

```diff
@@ -590,7 +591,7 @@ function transformUrlOptions(_object) {
   }
 
   if (object.readconcernlevel) {
-    object.readConcern = { level: object.readconcernlevel };
+    object.readConcern = new ReadConcern(object.readconcernlevel);
```

Comment:
should `ReadConcern.fromOptions` be updated to look for `readconcernlevel` as well?

Reply:
I don't think so because we should only ever receive a `readconcernlevel` from the URI. In all of the schemas from options management, we only support the `readConcern` object, not the `readConcernLevel` string.